br/br-log-architecture.md

```diff
@@ -0,0 +1,152 @@
+---
+title: TiDB Log Backup and PITR Architecture
+summary: Learn about the architecture of TiDB log backup and point-in-time recovery.
+---
+
+# TiDB Log Backup and PITR Architecture
+
+This document introduces the architecture and process of TiDB log backup and point-in-time recovery (PITR) using the br command-line tool (hereinafter referred to as `br`) as an example.
+
+## Architecture
+
+The log backup and PITR architecture is as follows:
+
+![BR log backup and PITR architecture](/media/br/br-log-arch.png)
+
+## Process of log backup
+
+The process of a cluster log backup is as follows:
+
+![BR log backup process design](/media/br/br-log-backup-ts.png)
+
+System components and key concepts involved in the log backup process:
+
+* **local metadata**: indicates the metadata backed up by a single TiKV node, including local checkpoint ts, global checkpoint ts, and backup file information.
+* **local checkpoint ts** (in local metadata): indicates that all logs generated before local checkpoint ts in this TiKV node have been backed up to the target storage.
+* **global checkpoint ts**: indicates that all logs generated before global checkpoint ts in all TiKV nodes have been backed up to the target storage. TiDB Coordinator calculates this timestamp by collecting local checkpoint ts of all TiKV node and then reports it to PD.
+* **TiDB Coordinator**: a TiDB node is elected as the coordinator, which is responsible for collecting and calculating the progress of the entire log backup task (global checkpoint ts). This component is stateless in design, and after its failure, a new Coordinator is elected from the surviving TiDB nodes.
+* **TiKV log backup observer**: runs on each TiKV node in the TiDB cluster, which is responsible for backing up log data. If a TiKV node fails, backing up the data range on it will be taken by other TiKV nodes after region re-election, and these nodes will redo back up data of the failure range starting from global checkpoint ts.
```

Comment:
```suggestion
* **TiKV log backup observer**: runs on each TiKV node in the TiDB cluster, which is responsible for backing up log data. If a TiKV node fails, backing up the data range on it will be taken by other TiKV nodes after region re-election, and these nodes will back up data of the failure range starting from global checkpoint ts.
```